hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -838,7 +839,7 @@ bool MDMParser::_powerOn(void)
     bool continue_cancel = false;
     bool retried_after_reset = false;
 
-    int i = 10;
+    int i = 25; // When modem not responsive on boot, AT/OK tried 25x (for ~30s) before hard reset
```

Comment:
Nit: could get red of these magic numbers and use constants since we are touching this, but can leave as-is.